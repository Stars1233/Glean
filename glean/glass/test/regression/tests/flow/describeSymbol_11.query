action: describeSymbol
args: "test/js/test/imports.js/s"
