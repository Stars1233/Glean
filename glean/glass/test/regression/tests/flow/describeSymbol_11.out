[
    "@generated",
    {
        "contains_relation": {
            "firstChild": "nondeterministic",
            "firstParent": "nondeterministic",
            "hasMoreChildren": false,
            "hasMoreParents": false
        },
        "extends_relation": {
            "firstChild": "nondeterministic",
            "firstParent": "nondeterministic",
            "hasMoreChildren": false,
            "hasMoreParents": false
        },
        "kind": 21,
        "language": 2,
        "location": {
            "filepath": "test/imports.js",
            "range": {
                "columnBegin": 5,
                "columnEnd": 19,
                "lineBegin": 37,
                "lineEnd": 37
            },
            "repository": "test"
        },
        "modifiers": [],
        "name": {
            "container": "imports",
            "localName": "s"
        },
        "pretty_comments": [],
        "repo_hash": "testhash",
        "signature": "s: string",
        "sym": "test/js/test/imports.js/s",
        "sym_location": {
            "filepath": "test/imports.js",
            "range": {
                "columnBegin": 5,
                "columnEnd": 19,
                "lineBegin": 37,
                "lineEnd": 37
            },
            "repository": "test"
        },
        "sym_other_locations": [],
        "type_xrefs": []
    }
]